modules/tests/photons_control_tests/test_clean.py

```diff
@@ -0,0 +1,155 @@
+# coding: spec
```

Comment:
interestingly these only pass because you set res_required=False on the SetHEV messages and don't have tests for the Gets. I'll have a look and see if I can make the mimic have basic hev support today or tomorrow and so you can write tests for your getters as well

Reply:
mimic supports the hev messages in the main branch now

Reply:
Ha, I have the mimic changes on another branch. I was still getting my head around it. :) Will look to see how close/far my attempt is from yours.

Reply:
Working out how to test that the configuration change "stuck" and that it would be used if needed was the last piece I wanted.